pkg/netpol/ingressanalyzer/ingress_analyzer.go

```diff
@@ -106,75 +125,151 @@ func convertServiceSelectorToLabelSelector(svcSelector map[string]string, svcStr
 	return selectorRes, nil
 }
 
-// //////////////////////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
in the function above this line `convertServiceSelectorToLabelSelector` , should return the `err` itself returned from `LabelSelectorAsSelector` (we don't want to miss/override the original error)